p2p/discover/table.go

```diff
@@ -46,7 +46,7 @@ const (
 	// We keep buckets for the upper 1/15 of distances because
 	// it's very unlikely we'll ever encounter a node that's closer.
 	hashBits          = len(common.Hash{}) * 8
-	nBuckets          = hashBits / 15       // Number of buckets
+	nBuckets          = hashBits            // Number of buckets
```

Comment:
I assume this is related to relaxing peer diversity for testing. If so, please make an issue to undo this.